xbmc/guilib/WindowIDs.dox

```diff
@@ -14,7 +14,6 @@ This page shows the window names, the window definition, the window ID and the s
 | FileManager             | WINDOW_FILES                         | 10003     | FileManager.xml                     |
 | Settings                | WINDOW_SETTINGS_MENU                 | 10004     | Settings.xml                        |
 | SystemInfo              | WINDOW_SYSTEM_INFORMATION            | 10007     | SettingsSystemInfo.xml              |
-| TestPattern             | WINDOW_TEST_PATTERN                  | 10008     | none                                |
```

Comment:
We should probably be starting to add this to the changelog
See https://github.com/xbmc/xbmc/pull/14542 as example

Reply:
That PR is not merged yet though, there's no changelog atm.